CryptoPkg/Library/OpensslLib/OpensslGen/include/openssl/opensslv.h

```diff
@@ -74,21 +74,21 @@ extern "C" {
  * longer variant with OPENSSL_VERSION_PRE_RELEASE_STR and
  * OPENSSL_VERSION_BUILD_METADATA_STR appended.
  */
-# define OPENSSL_VERSION_STR "3.4.1"
-# define OPENSSL_FULL_VERSION_STR "3.4.1"
+# define OPENSSL_VERSION_STR "3.5.0"
+# define OPENSSL_FULL_VERSION_STR "3.5.0-beta1"

```

Comment:
We should NOT use `beta1`.

Reply:
Not checked yet.

Reply:
Sure.  It's draft for a reason ;)

We'll have to wait for 3.5.1 for the openssl fixes mentioned in the description.